app/controllers/ConfigController.scala

```diff
@@ -56,34 +57,34 @@ class ConfigController @Inject() (implicit val env: Environment[User, SessionAut
+    val attributeCenterLat: Double = ConfigTable.getApiAttributeCenterLat
+    val attributeCenterLng: Double = ConfigTable.getApiAttributeCenterLng
+    val attributeZoom: Double = ConfigTable.getAttributeZoom
+    val attributeLat1: Double = ConfigTable.getAttributeLatOne
+    val attributeLng1: Double = ConfigTable.getAttributeLngOne
+    val attributeLat2: Double = ConfigTable.getAttributeLatTwo
+    val attributeLng2: Double = ConfigTable.getAttributeLngTwo
 
-    val streetCenterLat: Double = Play.configuration.getDouble("city-params.api-demos.street-center-lat." + cityStr).get
-    val streetCenterLng: Double = Play.configuration.getDouble("city-params.api-demos.street-center-lng." + cityStr).get
-    val streetZoom: Double = Play.configuration.getDouble("city-params.api-demos.street-zoom." + cityStr).get
-    val streetLat1: Double = Play.configuration.getDouble("city-params.api-demos.street-lat1." + cityStr).get
-    val streetLng1: Double = Play.configuration.getDouble("city-params.api-demos.street-lng1." + cityStr).get
-    val streetLat2: Double = Play.configuration.getDouble("city-params.api-demos.street-lat2." + cityStr).get
-    val streetLng2: Double = Play.configuration.getDouble("city-params.api-demos.street-lng2." + cityStr).get
+    val streetCenterLat: Double = ConfigTable.getStreetCenterLat
+    val streetCenterLng: Double = ConfigTable.getStreetCenterLng
+    val streetZoom: Double = ConfigTable.getStreetZoom
+    val streetLat1: Double = ConfigTable.getStreetLatOne
+    val streetLng1: Double = ConfigTable.getStreetLngOne
+    val streetLat2: Double = ConfigTable.getStreetLatTwo
+    val streetLng2: Double = ConfigTable.getStreetLngTwo
 
-    val regionCenterLat: Double = Play.configuration.getDouble("city-params.api-demos.region-center-lat." + cityStr).get
-    val regionCenterLng: Double = Play.configuration.getDouble("city-params.api-demos.region-center-lng." + cityStr).get
-    val regionZoom: Double = Play.configuration.getDouble("city-params.api-demos.region-zoom." + cityStr).get
-    val regionLat1: Double = Play.configuration.getDouble("city-params.api-demos.region-lat1." + cityStr).get
-    val regionLng1: Double = Play.configuration.getDouble("city-params.api-demos.region-lng1." + cityStr).get
-    val regionLat2: Double = Play.configuration.getDouble("city-params.api-demos.region-lat2." + cityStr).get
-    val regionLng2: Double = Play.configuration.getDouble("city-params.api-demos.region-lng2." + cityStr).get
+    val regionCenterLat: Double = ConfigTable.getRegionCenterLat
+    val regionCenterLng: Double = ConfigTable.getRegionCenterLng
+    val regionZoom: Double = ConfigTable.getRegionZoom
+    val regionLat1: Double = ConfigTable.getRegionLatOne
+    val regionLng1: Double = ConfigTable.getRegionLngOne
+    val regionLat2: Double = ConfigTable.getRegionLatTwo
+    val regionLng2: Double = ConfigTable.getRegionLngTwo
 
     Future.successful(Ok(Json.obj(
```

Comment:
Building on the comment above, you can add a `toJSON` function to the case class that will return a JSON object that looks like the ones below so that you can just call that function instead of building the JSON object for each of them.